model/ci/spack_intel.yaml

```diff
@@ -0,0 +1,20 @@
+spack:
```

Comment:
For consistency, shouldn't the existing spack.yaml be spac_gnu.yaml?    The gnu tests seem to be failing because the existing file does not have the: 
  concretizer:
    unify: true 
    for what gnu is using.

Reply:
Yep- I created spack_gnu.yaml, scrapped spack.yaml, added the same cmake change to gnu that I did to intel, and updated it to ubuntu-latest.